feat(ops): k8s

zacanger opened the pull request:
Adds Kubernetes setup. ~Marking as a draft~ (whoops, can't change from a regular PR to a draft) because I haven't tested it yet, and should add docs to the root README. If this isn't a useful addition, let me know; I'm probably going to use it myself, at least.

dessalines commented:
Sweet! I'll wait to merge it until you add the docs section to the main readme, mainly because I'm not sure how to run it.

zacanger commented:
Added docs! I also added a note about running databases in k8s: it works, but is considered to be a bad practice, probably at least partially because it's a PITA. I tested this briefly in Google Cloud.

dessalines commented:
Looks good, thanks!
